src/openrct2/object/SmallSceneryObject.cpp

```diff
@@ -119,11 +119,7 @@ void SmallSceneryObject::DrawPreview(rct_drawpixelinfo* dpi, int32_t width, int3
 
     if (_legacyType.HasFlag(SMALL_SCENERY_FLAG_HAS_GLASS))
     {
-        imageId = _legacyType.image + 0x44500004;
-        if (_legacyType.HasFlag(SMALL_SCENERY_FLAG_HAS_SECONDARY_COLOUR))
-        {
-            imageId |= 0x92000000;
-        }
```

Comment:
Why do we no longer require the branch

Reply:
I think it might be an implementation error. The scenery window does not do this. Additionally, the frame with the secondary colour is already drawn in the step before, and it seems glass always uses the primary colour.